src/kartikeyDJune20/AverageSumGrade.java

```diff
@@ -0,0 +1,57 @@
+package kartikeyDJune20;
+
+public class AverageSumGrade {
+	double sum(double frst, double second, double thrd) {
+		double sum = frst + second + thrd;
+		return sum;
+	}
+
+	double average(double first, double scnd, double thrd) {
+		double avg = 0;
+		if ((first > 100 || first < 0) || (scnd > 100 || scnd < 0) || (thrd > 100 || thrd < 0)) {
+			return avg;
+		} else {
+			sum(first, scnd, thrd);
+			avg = (first + scnd + thrd) / 3;
```

Comment:
here you should say avg = sum(first,scnd,thrd)/3

Reply:
Incororated the changes.. Thanks.. :-)